### ISSUE TITLE
[BUG] somd no longer reports energies in kcal.mol-1

### ISSUE THREAD

**Issue by jmichel80** [OpenBioSim/sire#79]
**Describe the bug**
Running a somd-freenrg calculation with the latest sire/devel produces the following output (excerpt)

```
(...)
###===========================================================###

Initial energy: 2.25653e+08 g Å^2 mol-1 ps-2
###=======================Minimisation========================###
Running minimization.
Tolerance for minimization: 1
Maximum number of minimization iterations: 1000
Energy after the minimization: -6.70368e+07 g Å^2 mol-1 ps-2
Energy minimization done.
###===========================================================###

###====================somd-freenrg run=======================###
Starting somd-freenrg run...
10000 moves 10 cycles, 400 ps simulation time
(...)
```
**Expected behavior**

The units of energy used to be in kcal.mol-1 which are more intuitive to work with 

To reproduce run the command below on the sample inputs attached, this should be reproducible with any somd input.

`somd-freenrg -C ./somd.cfg -p CUDA
`
[input.zip](https://github.com/OpenBioSim/sire/files/12052975/input.zip)

**Comment by chryswoods** [comment]
I've fixed this is the commit linked above. It is part of the AmberRst fix which will be merged into `main` (once conda is behaving) and that will included in `2023.3.1`.

The bug was that I'd switched to a new method of specifying the default names and default output formats for units, which I hadn't included in the old API. The values are correct - they are just printed in unintuitive base units.

**Comment by chryswoods** [comment]
The fix is now merged into `main` and `devel` :-)
